diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_de.ts lastfm-desktop-09012013-i18n/i18n/lastfm_de.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_de.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_de.ts	2013-01-10 02:14:25.450813737 +1100
@@ -893,11 +893,11 @@
         <translation>Empfohlen, weil du %1, %2, %3, %4 und %5 hörst.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>Aus der Musiksammlung von %1</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>Aus den Musiksammlungen von %1 und %2.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_en.ts lastfm-desktop-09012013-i18n/i18n/lastfm_en.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_en.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_en.ts	2013-01-10 02:14:50.265861580 +1100
@@ -893,7 +893,6 @@
         <translation>Recommended because you listen to %1, %2, %3, %4, and %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>From %1&apos;s library.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_es.ts lastfm-desktop-09012013-i18n/i18n/lastfm_es.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_es.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_es.ts	2013-01-10 02:15:03.993388049 +1100
@@ -893,11 +893,11 @@
         <translation>Recomendado porque escuchaste a %1, %2, %3, %4 y %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>De la colección de %1.</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>De las colecciones de %1 y %2.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_fr.ts lastfm-desktop-09012013-i18n/i18n/lastfm_fr.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_fr.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_fr.ts	2013-01-10 02:15:17.272913653 +1100
@@ -893,11 +893,11 @@
         <translation>Recommandé car vous écoutez %1, %2, %3, %4 et %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>Depuis la bibliothèque de %1.</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>Depuis les bibliothèques de %1 et %2.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_it.ts lastfm-desktop-09012013-i18n/i18n/lastfm_it.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_it.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_it.ts	2013-01-10 02:15:35.875449521 +1100
@@ -893,11 +893,11 @@
         <translation>Consigliato perché ascolti %1, %2, %3, %4 e %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>Dalla libreria di %1.</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>Dalle librerie di %1 e %2.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_ja.ts lastfm-desktop-09012013-i18n/i18n/lastfm_ja.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_ja.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_ja.ts	2013-01-10 02:15:50.683978073 +1100
@@ -892,7 +892,6 @@
         <source>From %1&apos;s library.</source>
         <translation>%1 さんのライブラリから。</translation>
     </message>
-    </message>
     <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_pl.ts lastfm-desktop-09012013-i18n/i18n/lastfm_pl.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_pl.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_pl.ts	2013-01-10 02:16:10.571516420 +1100
@@ -898,11 +898,11 @@
         <translation>Polecane ponieważ słuchałeś %1, %2, %3, %4 oraz %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>Z biblioteki %1.</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>Z bibliotek %1 oraz %2.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_pt.ts lastfm-desktop-09012013-i18n/i18n/lastfm_pt.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_pt.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_pt.ts	2013-01-10 02:16:38.171569636 +1100
@@ -897,7 +897,6 @@
         <source>From %1&apos;s library.</source>
         <translation>Da biblioteca de %1.</translation>
     </message>
-    </message>
     <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_sv.ts lastfm-desktop-09012013-i18n/i18n/lastfm_sv.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_sv.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_sv.ts	2013-01-10 02:16:59.183610150 +1100
@@ -893,11 +893,11 @@
         <translation>Rekommenderat eftersom du lyssnar på %1, %2, %3, %4 och %5.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>Från %1s bibliotek.</translation>
     </message>
+    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="672"/>
         <source>From %1 and %2&apos;s libraries.</source>
         <translation>Från %1 och %2s bibliotek.</translation>
diff -urN lastfm-desktop-09012013-orig/i18n/lastfm_tr.ts lastfm-desktop-09012013-i18n/i18n/lastfm_tr.ts
--- lastfm-desktop-09012013-orig/i18n/lastfm_tr.ts	2013-01-08 09:27:16.000000000 +1100
+++ lastfm-desktop-09012013-i18n/i18n/lastfm_tr.ts	2013-01-10 02:17:12.627636072 +1100
@@ -888,7 +888,6 @@
         <translation>%1, %2, %3, %4 ve %5 dinlediğiniz için önerildi.</translation>
     </message>
     <message>
-    <message>
         <location filename="../app/client/Widgets/MetadataWidget.cpp" line="671"/>
         <source>From %1&apos;s library.</source>
         <translation>%1 arşivinden.</translation>
